files/lib/system/exporter/WBB4xExporter.class.php

```diff
@@ -2181,6 +2246,16 @@ public function exportBlogEntries($offset, $limit)
         }
     }
 
+    private function getCoverPhotoPath(string $blogFilePath, array $row): string
```

Comment:
```suggestion
    private function getCoverPhotoPath(string $filePath, array $row): string
```
As the method is also used for the calendar events, it should not be called `blogFilePath`.